src/clr-angular/emphasis/alert/alert.ts

```diff
@@ -42,6 +43,28 @@ export class ClrAlert {
     return this.iconService.alertType;
   }
 
+  @Input('clrPolite') polite: boolean = true;
+  @Input('clrAssertive') assertive: boolean;
+  @Input('clrOff') off: boolean;
+  /**
+   * There is clear order on how the attributes will take effect.
+   * Assertive, Off, Polite.
+   *
+   * Polite is default if non is passed.
+   *
+   * In the case of setting all of them to true. Assertive will be used.
+   *
+   */
+  get setAriaLive(): string {
+    if (isBooleanAttributeSet(this.assertive)) {
+      return 'assertive';
+    }
+    if (isBooleanAttributeSet(this.off)) {
```

Comment:
Right now, this appears to be the only tested conditional path in `setAriaLive`. We need more tests.

Reply:
You are right :) on it...